app/controllers/region_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RegionController < ApplicationController
+  before_filter :load_vars
+
+  def show
+  end
+
+  private
+
+  def load_vars
+    @region = Region.where(iso: params[:iso].upcase).first
+
```

Comment:
Oh god please get rid of the blank lines

Reply:
Thanks @spencerldixon I have removed those blank lines.